terraform/install.sh

```diff
@@ -58,6 +60,51 @@ getBillingAccount() {
   log "using billing account: $billing_acct"
 }
 
+getProject() {
+  log "Checking for project list"
+  found_projects=$(gcloud projects list --filter="name:stackdriver-sandbox-*" --format="value(projectId)")
+  if [ -z "$found_projects" ] || [[ ${#found_projects[@]} -eq 0 ]]; then
+    create_flag=true
+  else
+      log "Which project would you like to use?:"
+      IFS_bak=$IFS
+      IFS=$'\n'
+      select opt in ${found_projects} "create" "cancel"; do
+        if [[ "${opt}" == "cancel" ]]; then
+          exit 0
+        elif [[ "${opt}" == "create" ]]; then
+          create_flag=true
+          log "create a new Sandbox!"
+          break
+        elif [[ -z "${opt}" ]]; then
+          log "invalid response"
+        else
+          project_id=$opt
+          bucket_name="$project_id-bucket"
+          break
+        fi
+      done
+      IFS=$IFS_bak
+  fi
+}
+
+createProject() {
+  if [ "$create_flag" = true ]; then
+    # generate random id
+    project_id="stackdriver-sandbox-$(od -N 4 -t uL -An /dev/urandom | tr -d " ")"
+    bucket_name="$project_id-bucket"
```

Comment:
I suggest that we name it so it's clear what's the purpose of the bucket. You don't need to include the projectid in the name since each bucket belongs to a project.
You can name it stackdriver-sandbox-state or something along those lines.

Reply:
Ok.